tests/pl/homeassistant_HassTurnOff.yaml

```diff
@@ -3,38 +3,44 @@ tests:
   - sentences:
       - Wyłącz proszę lampkę nocną
       - Zgaś lampkę nocną
+      - Wyłącz lampkę nocną
     intent:
       name: HassTurnOff
       slots:
         name: light.bedroom_lamp
+    response: Wyłączono lampkę nocną
   - sentences:
       - Wyłącz proszę wentylator sufitowy
       - Zatrzymaj wentylator sufitowy
     intent:
       name: HassTurnOff
       slots:
         name: fan.ceiling
+    response: Wyłączono wentylator sufitowy
   - sentences:
       - Zgaś lampę nocną
-      - Wyłącz lampkę nocną
     intent:
       name: HassTurnOff
       slots:
         name: light.bedroom_lamp
+    response: Wyłączono lampę nocną
   - sentences:
       - Zamknij roletę
     intent:
       name: HassTurnOff
       slots:
         name: cover.roleta
+    response: Zamknięto roletę
   - sentences:
       - Zamknij roletę w salonie
     intent:
       name: HassTurnOff
       slots:
         name: cover.roleta
         area: living_room
+    response: Zamknięto rolety w salonie
   - sentences:
+      - Zamknij garaż
```

Comment:
Hmm, ciekawe, bo nic z tego nie działa, a testy przechodzi. Wiadomo czemu?

Reply:
I've answered in another comment, continue there.